line_beam_parameters.py

```diff
@@ -27,21 +29,38 @@ class LineBeamParametersControl(Display):
     _setting_bits = False
     energy_channel = None
 
+    # Monitor the rate readback for use in the zero rate button
+    rate_channel = None
+
+    # Signal to set a new rate from the combobox or from zero rate button
+    update_rate_signal = QtCore.Signal(int)
+
+
```

Comment:
Style off here? Is flake8 being run?

Reply:
https://github.com/pcdshub/pmps-ui/blob/97ddee40b63a72e4543481e5986eb0bc16829e4c/.pre-commit-config.yaml#L20-L24